docs/creating-editing-and-removing-specialist-document-types-and-finders.md

```diff
@@ -18,6 +18,7 @@ Recent changes to Specialist Publisher now allow users to self-serve. These requ
 See [example PR here](https://github.com/alphagov/publishing-api/pull/3026/files).
 
 1. Add the format to [allowed document types list](https://github.com/alphagov/publishing-api/blob/main/content_schemas/allowed_document_types.yml).
+Ideally, the schema name should align with the format name for consistency. While this isn’t always possible, a mismatch may lead to failures in specialist publisher's automated tests. In such cases, you may need to mark your finder as an [exception](https://github.com/alphagov/specialist-publisher/blob/main/spec/models/document_type_spec.rb#L6) to general testing. 
```

Comment:
While I remember, can you write up a Trello card(s) outlining the exceptions, so that we can take a look at them one by one and figure out a way of genericising the tests?